### PR TITLE
auth status is fails to give error on expiration bugzilla 1903693

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
Based on the Bugzilla 1903693 it is failing on the first occurrence of auth status, it was passing earlier in the test because after any API call it starts passing as an expected message. I have to change the assertion location to validate this Bugzilla. We will hold this PR till Bugzilla get valid confirmation

### CODE REVIEW

**Review by JacobCallahan — APPROVED** [review]
Changes are simple enough. What's your reason for the do not merge label?

**Comment by omkarkhatavkar** [comment]
@JacobCallahan Just waiting for the response from Dev in the Bugzilla, Personally I think it is a valid bugzilla.

**Review by mshriver — APPROVED** [review]

File: tests/foreman/cli/test_auth.py
Comment (mshriver) [inline tests/foreman/cli/test_auth.py:301]: The BZ has been triaged, but doesn't look like it has been targeted for any release yet.

Since adding the BZ to the metadata will cause the testcase to skip execution if the bug is open, I think it would be a better approach here to execute this `with` block based on the BZ status.  There are examples in the framework of doing this - you just build an `if` statement that checks the BZ blocking status.

Then, until this BZ is fixed, the test can still run, and just won't check that a CLIReturnCodeError is raised when you try to list orgs without a valid token.

**Comment by mshriver** [comment]
Moving this to draft state to clearly show it is not waiting review or action by maintainers, but an action by the author.

**Review by mshriver — APPROVED** [review]

**Pull request merged**
